### ISSUE TITLE
Firefox add-on is outdated?

### ISSUE THREAD

**Issue by FirewallDaProtogen** [samuelmaddock/metastream#456]
Steps to Reproduce:

1. go to https://addons.mozilla.org/en-US/firefox/addon/metastream-remote/
2. look at version

**Comment by rugk** [comment]
I agree, this does not seem to work with new versions too. Chrome works fine, Firefox does not (likely due to this issue?)? Tested with https://www.ardmediathek.de/ and some direct MP4 file from a video. @samuelmaddock

**Comment by rivenirvana** [comment]
I think this is causing some problems when watching with someone on a different browser (Chrome).
